### ISSUE TITLE
Reverse screenshot order in dashboard

### ISSUE THREAD

**Issue by dcastil** [cypress-io/cypress#1638]
### Versions  
Cypress v2.1.0
Chrome v65
Mac OS v10.13.4

**Comment by brian-mann** [comment]
@jennifer-shehane and @bahmutov can you look at this please?

**Comment by jennifer-shehane** [comment]
Yes. I noticed this when working on our 3.0 release and [requested that the api reverse the order](https://github.com/cypress-io/cypress-api/issues/430), but it was pushed back as a decision for the front-end.

I can go ahead and reverse the order in the dashboard and get a quick fix out hopefully today. https://github.com/cypress-io/cypress-dashboard/issues/665

**Comment by bahmutov** [comment]
yeah, once 3.0 comes out, the screenshot order should be correct per spec / test

**Comment by jennifer-shehane** [comment]
Got a PR open for this https://github.com/cypress-io/cypress-dashboard/pull/666

**Comment by jennifer-shehane** [comment]
This should be fixed now.

**Comment by dcastil** [comment]
Now when I switch between the tabs, the order of the screenshots changes. So when I am on the page of a CI build in the Cypress dashboard and click on "Screenshots", they are ordered in chronological order. But when I click on "Output" and then "Screenshots" again, the screenshots are in reversed chronological order. 😄
